openid-sharedsignals-framework-1_0.md

```diff
@@ -1466,6 +1466,10 @@ stream_id
 On receiving a valid request, the Event Transmitter responds with a 200 OK
 response containing a [JSON][RFC7159] object with the following attributes:
 
+stream_id
+
+> REQUIRED. The stream whose status is being queried.
+
```

Comment:
nit: to add more clarity should we say

> The identifier of the stream whose status is being queried

and change it throughout the spec

Reply:
Good catch